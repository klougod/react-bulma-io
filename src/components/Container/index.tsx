import React from 'react'
import PropTypes from 'prop-types'

export const Container = ({ className, children }: any) => (
  <div className={`container ${className}`}>{children}</div>
)

Container.defaultProps = {
  className: ''
}

Container.propTypes = {
  className: PropTypes.string,
  children: PropTypes.node
}

export default Container
