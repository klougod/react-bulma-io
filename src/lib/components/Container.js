import React from 'react'
import PropTypes from 'prop-types'

export const Container = ({ className, children }) => (
  <div className={`container ${className}`}>
    {children}
  </div>
)

Container.defaultProps = {
  className: ''
}

Container.propTypes = {
  className: PropTypes.string,
  children: PropTypes.node.isRequired
}

export default Container