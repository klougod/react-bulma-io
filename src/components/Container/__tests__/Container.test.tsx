import React from 'react'
import Enzyme from 'enzyme'

import Container from '../'

describe('Container component test', function () {
  it('should render without crash without props', function () {
    const containerMock = Enzyme.shallow(<Container />)
    expect(containerMock).toMatchSnapshot()
  })

  it('should render without crash with props', function () {
    const containerMockProps = {
      className: 'containerCLass',
    }
    const containerMock = Enzyme.shallow(<Container {...containerMockProps} />)
    expect(containerMock).toMatchSnapshot()
  })
})
