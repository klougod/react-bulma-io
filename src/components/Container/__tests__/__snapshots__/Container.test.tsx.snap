// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component test should render without crash with props 1`] = `
<div
  className="container containerCLass"
/>
`;

exports[`Container component test should render without crash without props 1`] = `
<div
  className="container "
/>
`;
